From 4f80d3b1851573bf8545768e5ddb4eb57557fbab Mon Sep 17 00:00:00 2001
From: Oliver Nightingale <oliven@amazon.com>
Date: Wed, 15 Jan 2020 07:28:17 +0000
Subject: [PATCH] Add some quality of life improvements to tmux

* reload config with prefix R
* open vertical split with prefix v
* toggle synchronise panes with prefix y
---
 tmux.conf | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/tmux.conf b/tmux.conf
index d8e5b65..5f2ac82 100644
--- a/tmux.conf
+++ b/tmux.conf
@@ -40,5 +40,14 @@ bind g split-window -h -c "#{pane_current_path}" "tig"
 # resize windows
 bind = select-layout even-horizontal

+# open a vertical split
+bind v split-window -h
+
+# toggle synchronize panes
+bind y setw synchronize-panes
+
+# reload tmux config
+bind-key R source-file ~/.tmux.conf \; display-message "~/.tmux.conf reloaded"
+
 run-shell ". ~/.tmux/color.tmux"

--
2.17.1

